Jenkinsfile

```diff
@@ -890,7 +979,7 @@ pipeline {
                     }
                     agent {
                         // 8+ node cluster with 1 IB/node + 1 test control node
-                        label 'ci_nvme9'
+                        label params.CI_NVME_9_LABEL
```

Comment:
It would be nice if these could be commit pragmas also.

Reply:
I would have to do some tests to see what works for the label string.

Reply:
I was thinking something similar, where it could be useful for these params to be commit pragmas, and vice versa. Is it feasible to someone integrate the params and pragmas to avoid duplication?